.dockerignore

```diff
@@ -8,3 +8,7 @@ docker-cache
 arm-compile.Dockerfile
 Dockerfile
 .github/
+docs/
+wiki/
+.vscode
+*.md
```

Comment:
This excludes unnecessary files in the `COPY . .` stage of the docker build. Hopefully, when there is only a change to the docs the docker build for the `dev` tag will be completely cached.